src/txn/txn_rollback_to_stable.c

```diff
@@ -421,7 +421,11 @@ __rollback_abort_row_ondisk_kv(
     __wt_row_leaf_value_cell(session, page, rip, NULL, vpack);
     prepared = vpack->tw.prepare;
     if (WT_IS_HS(S2BT(session))) {
-        if (vpack->tw.durable_stop_ts > rollback_timestamp) {
+        /*
+         * Abort the history store update with stop durable timestamp greater than stable timestamp
+         * or the updates with max stop timestamp.
```

Comment:
Could you please expand this comment, adding "which implies that they are associated with prepared transactions" to the end?

Reply:
Thanks, fixed.